testsuite/tests/letrec/underscore_binding.ml

```diff
@@ -0,0 +1,28 @@
+(**************************************************************************)
+(*                                                                        *)
+(*                                OCaml                                   *)
+(*                                                                        *)
+(*           Gabriel Scherer, projet Gallium, INRIA Rocquencourt          *)
+(*                                                                        *)
+(*   Copyright 2016 Institut National de Recherche en Informatique et     *)
+(*     en Automatique.                                                    *)
+(*                                                                        *)
+(*   All rights reserved.  This file is distributed under the terms of    *)
+(*   the GNU Lesser General Public License version 2.1, with the          *)
+(*   special exception on linking described in the file LICENSE.          *)
+(*                                                                        *)
+(**************************************************************************)
```

Comment:
Please don't use headers in testsuite files. Especially headers with the wrong names...